packages/syft/src/syft/service/worker/utils.py

```diff
@@ -82,21 +81,36 @@ def extract_config_from_backend(
     environment = details["Config"]["Env"]
 
     # Extract Volume Binds
+    vol_binds = {}
+
+    # ignore any irrelevant binds for the worker like
+    # packages/grid/backend/grid:/root/app/grid
+    # packages/syft:/root/app/syft
+    # packages/grid/data/package-cache:/root/.cache
+    valid_binds = {
+        "/var/run/docker.sock",
+        "/root/.cache",
+    }
+
     for vol in host_config["Binds"]:
         parts = vol.split(":")
         key = parts[0]
         bind = parts[1]
         mode = parts[2]
-        if "/storage" in bind:
+
+        if "/root/data/creds" in vol:
             # we need this because otherwise we are using the same node private key
             # which will make account creation fail
-            worker_postfix = worker_name.split("-", 1)[1]
-            key = f"{key}-{worker_postfix}"
-        extracted_config["volume_binds"][key] = {"bind": bind, "mode": mode}
+            key = f"{key}-{worker_name}"
```

Comment:
There is an issue in WSL2 here.
- The DEV_MODE=True mount is `./path/to/local/creds:/root/data/creds` (when mounted becomes absolute - /home/user/dir/path/to/local/creds`)
- This is available on Linux, but docker desktop windows fails to resolve this path as it's not a windows path and it mounts against the docker-desktop-data WSL2 distro and not the distro it was invoked from. This makes syft function call fails - probably because credentials sharing not working as expected?

@shubham3121

Reply:
Nice catch @yashgorana. We can think about mounting according to os type, but again since we will be kubernetes for now, not sure if this is an urgent fix.

Reply:
yeah not an urgent fix